### ISSUE TITLE
[FEATURE] Ability to pass connect_args to FastAPISessionMaker

### ISSUE THREAD

**Issue by jordan-dimov** [dmontagu/fastapi-utils#174]
I need to be able to pass `connect_args` to FastAPISessionMaker so I can configure the SQLAlchemy engine to correctly connect to an AWS Aurora cluster (which needs `aurora_cluster_arn` and `secret_arn` to be passed as `connect_args`). 

At the moment, I'm not able to do this, because there is no easy way to customise the `get_engine` function which FastAPISessionMaker uses.

**Comment by jkklapp** [comment]
Perhaps I'm lacking some context here, but you could create your custom `SQLAlchemy` engine and pass it to `FastAPISessionMaker` as its `_cached_engine`. Would this help?

**Comment by jordan-dimov** [comment]
Yes, thank you, this could work, although elsewhere it was recommended to not create a global engine object. Would be good if the documentation provides a recommended way of doing this.

**Comment by jkklapp** [comment]
Granted, setting what it seems to be a private variable seems like a hack to me, so I understand that not being documented.

If the maintainers (@dmontagu) agree I can work on this.
